doc/nrf/images/aws_fota_dfu_sequence.uml.txt

```diff
@@ -2,50 +2,65 @@ title AWS Firmware-over-the-air
 participant Device
```

Comment:
Remember to also generate the .svg file with   https://sequencediagram.org

Reply:
Right, the .svg file should now be updated.